src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/DoNotDecreaseInheritedMemberVisibilityTests.Fixer.cs

```diff
@@ -12,22 +11,387 @@ public class DoNotDecreaseInheritedMemberVisibilityFixerTests : CodeFixTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicDoNotDecreaseInheritedMemberVisibilityAnalyzer();
+            return new DoNotDecreaseInheritedMemberVisibilityAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpDoNotDecreaseInheritedMemberVisibilityAnalyzer();
+            return new DoNotDecreaseInheritedMemberVisibilityAnalyzer();
         }
 
         protected override CodeFixProvider GetBasicCodeFixProvider()
         {
-            return new BasicDoNotDecreaseInheritedMemberVisibilityFixer();
+            return new DoNotDecreaseInheritedMemberVisibilityFixer();
         }
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new CSharpDoNotDecreaseInheritedMemberVisibilityFixer();
+            return new DoNotDecreaseInheritedMemberVisibilityFixer();
+        }
+
+        [Fact]
+        public void DecreaseCSharpMemberVisibility()
```

Comment:
@srivatsn Any guidance on test naming conventions? I know a lot of them put the language name first (`CSharpFooBar`, `BasicFooBar`).

(Maybe I should add that to the TODO list for the "analyzer ecosystem".)